### ISSUE TITLE
Exporting wrong XDG envs

### ISSUE THREAD

**Issue by rodweb** [doron-cohen/antidot#116]
Although I have set my envs, they are being reexported in a wrong way. I'm running Zsh.
```
❯ echo $XDG_CACHE_HOME
/home/rod/.cache
❯ eval "$(antidot init)"
❯ echo $XDG_CACHE_HOME
{XDG_CACHE_HOME:-/home/rod/.cache}
❯ antidot --version
antidot version 0.6.1
```

```
❯ pwd
/home/rod/{XDG_DATA_HOME:-/home/rod/.local
```

**Comment by doron-cohen** [comment]
Thanks @rodweb . Could you add the output of `antidot init`? Also `env` output could help. Just make sure it doesn't contain anything sensitive like a password or token.

**Comment by rodweb** [comment]
Sure.

```
❯ antidot init
export XDG_DATA_HOME="{XDG_DATA_HOME:-/home/rod/.local/share}"
export XDG_CONFIG_HOME="{XDG_CONFIG_HOME:-/home/rod/.config}"
export XDG_CACHE_HOME="{XDG_CACHE_HOME:-/home/rod/.cache}"

if [ -f "/home/rod/.local/share/antidot/env.sh" ]; then source "/home/rod/.local/share/antidot/env.sh"; fi
if [ -f "/home/rod/.local/share/antidot/alias.sh" ]; then source "/home/rod/.local/share/antidot/alias.sh"; fi
```

Before:
```
❯ env
ALACRITTY_LOG=/tmp/Alacritty-1314.log
ASDF_DATA_DIR=/home/rod/.config/asdf
ASDF_DIR=/opt/asdf-vm
AWS_CONFIG_FILE=/home/rod/.config/aws/config
AWS_SHARED_CREDENTIALS_FILE=/home/rod/.config/aws/credentials
COLORTERM=truecolor
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
DISPLAY=:0
EDITOR=nvim
FZF_DEFAULT_COMMAND=fd --type f
GNUPGHOME=/home/rod/.local/share/gnupg
GOPATH=/home/rod/dev/go
HISTFILE=/home/rod/.local/share/zsh/history
HISTSIZE=1000
HOME=/home/rod
INPUTRC=/home/rod/.config/readline/inputrc
INVOCATION_ID=f1053c55d6564fd19a5352f3cb2e13d0
JOURNAL_STREAM=8:21271
KEYCHAIN_DIR=/run/user/1000/keychain
LANG=en_US.UTF-8
LESSHISTFILE=/home/rod/.cache/less/history
LINES=42
LOGNAME=rod
MAIL=/var/spool/mail/rod
MANPAGER=sh -c 'col -bx | bat -l man -p'
MOTD_SHOWN=pam
NODE_REPL_HISTORY=/home/rod/.local/share/nodejs/repl_history
OLDPWD=/home/rod
P9K_SSH=0
P9K_TTY=old
PASSWORD_STORE_DIR=/home/rod/.local/share/pass
PATH=/home/rod/bin:/home/rod/bin:/home/rod/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/home/rod/dev/go/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/rod/dev/go/bin:/opt/asdf-vm/bin:/home/rod/dev/go/bin:/opt/asdf-vm/bin
PWD=/home/rod
SAVEHIST=100000
SHELL=/bin/zsh
SHLVL=3
SSH_AGENT_PID=1223
SSH_ASKPASS=ssh-askpass
SSH_AUTH_SOCK=/tmp/ssh-iluzhRshuAo0/agent.1222
TERM=alacritty
USER=rod
VISUAL=nvim
WINDOWID=23068674
WINDOWPATH=1
XAUTHORITY=/home/rod/.Xauthority
XDG_CACHE_HOME=/home/rod/.cache
XDG_CONFIG_HOME=/home/rod/.config
XDG_DATA_HOME=/home/rod/.local/share
XDG_RUNTIME_DIR=/run/user/1000
XDG_SEAT=seat0
XDG_SESSION_CLASS=user
XDG_SESSION_ID=1
XDG_SESSION_TYPE=tty
XDG_VTNR=1
ZDOTDIR=/home/rod/.config/zsh
_JAVA_AWT_WM_NONREPARENTING=1
_JAVA_OPTIONS=-Djava.util.prefs.userRoot=/home/rod/.config/java
_P9K_TTY=/dev/pts/0
COLUMNS=160
_=/usr/bin/env
```

After:
```
ALACRITTY_LOG=/tmp/Alacritty-1314.log
ASDF_DATA_DIR=/home/rod/.config/asdf
ASDF_DIR=/opt/asdf-vm
AWS_CONFIG_FILE=/home/rod/.config/aws/config
AWS_SHARED_CREDENTIALS_FILE=/home/rod/.config/aws/credentials
COLORTERM=truecolor
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
DISPLAY=:0
EDITOR=nvim
FZF_DEFAULT_COMMAND=fd --type f
GNUPGHOME=/home/rod/.local/share/gnupg
GOPATH=/home/rod/dev/go
HISTFILE=/home/rod/.local/share/zsh/history
HISTSIZE=1000
HOME=/home/rod
INPUTRC=/home/rod/.config/readline/inputrc
INVOCATION_ID=f1053c55d6564fd19a5352f3cb2e13d0
JOURNAL_STREAM=8:21271
KEYCHAIN_DIR=/run/user/1000/keychain
LANG=en_US.UTF-8
LESSHISTFILE=/home/rod/.cache/less/history
LINES=42
LOGNAME=rod
MAIL=/var/spool/mail/rod
MANPAGER=sh -c 'col -bx | bat -l man -p'
MOTD_SHOWN=pam
NODE_REPL_HISTORY=/home/rod/.local/share/nodejs/repl_history
OLDPWD=/home/rod
P9K_SSH=0
P9K_TTY=old
PASSWORD_STORE_DIR=/home/rod/.local/share/pass
PATH=/home/rod/bin:/home/rod/bin:/home/rod/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/home/rod/dev/go/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/rod/dev/go/bin:/opt/asdf-vm/bin:/home/rod/dev/go/bin:/opt/asdf-vm/bin
PWD=/home/rod
SAVEHIST=100000
SHELL=/bin/zsh
SHLVL=3
SSH_AGENT_PID=1223
SSH_ASKPASS=ssh-askpass
SSH_AUTH_SOCK=/tmp/ssh-iluzhRshuAo0/agent.1222
TERM=alacritty
USER=rod
VISUAL=nvim
WINDOWID=23068674
WINDOWPATH=1
XAUTHORITY=/home/rod/.Xauthority
XDG_CACHE_HOME={XDG_CACHE_HOME:-/home/rod/.cache}
XDG_CONFIG_HOME={XDG_CONFIG_HOME:-/home/rod/.config}
XDG_DATA_HOME={XDG_DATA_HOME:-/home/rod/.local/share}
XDG_RUNTIME_DIR=/run/user/1000
XDG_SEAT=seat0
XDG_SESSION_CLASS=user
XDG_SESSION_ID=1
XDG_SESSION_TYPE=tty
XDG_VTNR=1
ZDOTDIR=/home/rod/.config/zsh
_JAVA_AWT_WM_NONREPARENTING=1
_JAVA_OPTIONS=-Djava.util.prefs.userRoot=/home/rod/.config/java
_P9K_TTY=/dev/pts/0
COLUMNS=160
PYLINTHOME={XDG_CACHE_HOME:-/home/rod/.cache}/pylint
npm_config_cache={XDG_CACHE_HOME:-/home/rod/.cache}/npm
_=/usr/bin/env
```

**Comment by doron-cohen** [comment]
What a dumb typo. Fixing

**Comment by doron-cohen** [comment]
This is now fixed in [0.6.2](https://github.com/doron-cohen/antidot/releases/tag/v0.6.2).

**Comment by rodweb** [comment]
@doron-cohen Thank you!
